client/my-sites/checkout/src/hooks/use-stored-payment-methods.tsx

```diff
@@ -81,6 +81,7 @@ export function useStoredPaymentMethods( {
 		queryKey,
 		queryFn: () => fetchPaymentMethods( type, expired ),
 		enabled: ! isLoggedOut,
+		staleTime: 5 * 60 * 1000, // 5 minutes
```

Comment:
Added this because this hook is now being used multiple times in different places and we can avoid making extra network calls. Happy to discuss this change!

Reply:
> I'll revert this line.

Thank you. I think that's safer.

> component also calls the useStoredPaymentMethods hook, which would make another network request.

Not necessarily. `useQuery` automatically caches data and does not repeat queries unless it considers them stale. While we could adjust the logic it uses to determine that, I think it's best that any experimentation there be done separately so we can test all the places that use it.

Reply:
The default behaviour is to consider all cached data as stale: https://tanstack.com/query/v4/docs/react/guides/important-defaults

So even though the data is cached, it is considered "stale" as there is no `staleTime` specified, and the network request will be repeated.